docs/reference/commandline/swarm_leave.md

```diff
@@ -16,13 +16,15 @@ Usage:  docker swarm leave [OPTIONS]
 Leave a swarm
 
 Options:
-      --force   Force leave ignoring warnings.
+      --force   Force a node to leave the swarm, ignoring warnings.
       --help    Print usage
 ```
 
-This command causes the node to leave the swarm.
+This command causes the node to leave the swarm. When used on a manager,
+the swarm may lose quorum. To prevent this, demote a mananger to a worker
```

Comment:
The swarm can lose quorum if you demote a manager to worker, too. So it does not "prevent" losing quorum. To prevent that you need to a) join a new manager or b) promote a node to manager